lua/telescope/mappings.lua

```diff
@@ -148,9 +148,13 @@ mappings.default_mappings = config.values.default_mappings
 
       ["<C-u>"] = actions.preview_scrolling_up,
       ["<C-d>"] = actions.preview_scrolling_down,
+      ["<C-f>"] = actions.preview_scrolling_left,
+      ["<C-k>"] = actions.preview_scrolling_right,
 
       ["<PageUp>"] = actions.results_scrolling_up,
       ["<PageDown>"] = actions.results_scrolling_down,
+      ["<M-f>"] = actions.results_scrolling_left,
+      ["<M-k>"] = actions.results_scrolling_right,
```

Comment:
i am usually not a fan of introducing new mappings, but we could do these because its a general feature.

You you add docs for these in the readme? after that i think this is good to go. Thanks for implementing it :)

Reply:
Added README.md docs in df536ce!